pkg/code/data/nonce/tests/tests.go

```diff
@@ -260,8 +310,9 @@ func testGetRandomAvailableByPurpose(t *testing.T, s nonce.Store) {
 				nonce.StateUnknown,
 				nonce.StateAvailable,
 				nonce.StateReserved,
+				nonce.StateClaimed,
 			} {
-				for i := 0; i < 500; i++ {
+				for i := 0; i < 50; i++ {
```

Comment:
Shrunk this in size to fit `Available, Claimed, Claimed(Expired)` within the `LIMIT` window.